internals/secrethub/run_test.go

```diff
@@ -82,6 +82,36 @@ func TestParseEnv(t *testing.T) {
 				},
 			},
 		},
+		"success with single quotes": {
+			raw: "key='value'",
+			expected: []envvar{
+				{
+					key:        "key",
+					value:      "value",
+					lineNumber: 1,
+				},
+			},
+		},
+		"success with double quotes": {
+			raw: `key="value"`,
+			expected: []envvar{
+				{
+					key:        "key",
+					value:      "value",
+					lineNumber: 1,
+				},
+			},
+		},
+		"success with quotes and whitespace": {
+			raw: "key = 'value'",
+			expected: []envvar{
+				{
+					key:        "key",
+					value:      "value",
+					lineNumber: 1,
+				},
+			},
+		},
```

Comment:
Could you add a testcase for maintaining quotes within the value (e.g. `key={"foo":"bar"}`)?